src/System.IO.Pipelines/PipeFactory.cs

```diff
@@ -49,9 +49,9 @@ public IPipeReader CreateReader(Stream stream)
             return pipe;
         }
 
-        private async void ExecuteCopyToAsync(Pipe pipe, Stream stream)
+        private Task ExecuteCopyToAsync(Pipe pipe, Stream stream)
         {
-            await stream.CopyToAsync(pipe);
```

Comment:
ContinueWith() can also be confusing because that in and of itself doesn't observe the task. The Task.Exception property needs to be accessed. Look below for an example of this confusion.

ContinueWith() can die in a 🔥 try/catch is clearly superior. We have multiple examples in one PR already of ContinueWith() being done wrong.